saleor/graphql/webhook/subscription_types.py

```diff
@@ -38,6 +43,133 @@
+            WebhookEventAsyncType.SALE_UPDATED: SaleUpdated,
+            WebhookEventAsyncType.SALE_DELETED: SaleDeleted,
+            WebhookEventAsyncType.INVOICE_REQUESTED: InvoiceRequested,
+            WebhookEventAsyncType.INVOICE_DELETED: InvoiceDeleted,
+            WebhookEventAsyncType.INVOICE_SENT: InvoiceSent,
+            WebhookEventAsyncType.FULFILLMENT_CREATED: FulfillmentCreated,
+            WebhookEventAsyncType.FULFILLMENT_CANCELED: FulfillmentCanceled,
+            WebhookEventAsyncType.CUSTOMER_CREATED: CustomerCreated,
+            WebhookEventAsyncType.CUSTOMER_UPDATED: CustomerUpdated,
+            WebhookEventAsyncType.COLLECTION_CREATED: CollectionCreated,
+            WebhookEventAsyncType.COLLECTION_UPDATED: CollectionUpdated,
+            WebhookEventAsyncType.COLLECTION_DELETED: CollectionDeleted,
+            WebhookEventAsyncType.CHECKOUT_CREATED: CheckoutCreated,
+            WebhookEventAsyncType.CHECKOUT_UPDATED: CheckoutUpdated,
+            WebhookEventAsyncType.PAGE_CREATED: PageCreated,
+            WebhookEventAsyncType.PAGE_UPDATED: PageUpdated,
+            WebhookEventAsyncType.PAGE_DELETED: PageDeleted,
+            WebhookEventAsyncType.SHIPPING_PRICE_CREATED: ShippingPriceCreated,
+            WebhookEventAsyncType.SHIPPING_PRICE_UPDATED: ShippingPriceUpdated,
+            WebhookEventAsyncType.SHIPPING_PRICE_DELETED: ShippingPriceDeleted,
+            WebhookEventAsyncType.SHIPPING_ZONE_CREATED: ShippingZoneCreated,
+            WebhookEventAsyncType.SHIPPING_ZONE_UPDATED: ShippingZoneUpdated,
+            WebhookEventAsyncType.SHIPPING_ZONE_DELETED: ShippingZoneDeleted,
+            WebhookEventAsyncType.TRANSACTION_ACTION_REQUEST: TransactionActionRequest,
+            WebhookEventAsyncType.TRANSLATION_CREATED: TranslationCreated,
+            WebhookEventAsyncType.TRANSLATION_UPDATED: TranslationUpdated,
+            WebhookEventAsyncType.VOUCHER_CREATED: VoucherCreated,
+            WebhookEventAsyncType.VOUCHER_UPDATED: VoucherUpdated,
+            WebhookEventAsyncType.VOUCHER_DELETED: VoucherDeleted,
+            WebhookEventAsyncType.WAREHOUSE_CREATED: WarehouseCreated,
+            WebhookEventAsyncType.WAREHOUSE_UPDATED: WarehouseUpdated,
+            WebhookEventAsyncType.WAREHOUSE_DELETED: WarehouseDeleted,
+        }
+        return types.get(object_type)
+
+    @classmethod
+    def resolve_type(cls, instance, info):
+        type_str, _ = instance
+        return cls.get_type(type_str)
+
+    @staticmethod
+    def resolve_issued_at(_root, _info):
+        return timezone.now()
+
+    @staticmethod
+    def resolve_version(_root, _info):
+        return __version__
+
+    @staticmethod
+    def resolve_recipient(_root, info):
+        return info.context.app
+
+    @staticmethod
+    def resolve_issuing_principal(_root, info):
+        if not info.context.requestor:
+            return None
+        return info.context.requestor
```

Comment:
Cannot be just:
```suggestion
        return info.context.requestor
```
?